### PR TITLE
feat: Approval management setup in personal workstation

### PR DESCRIPTION
**Pull request by shuofan**
#### What this PR does / why we need it:
Approval management setup in personal workstation

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/gantt?filter__urlQuery=eyJpdGVyYXRpb25JRHMiOlsxMDkwLDExMTUsMTA2OSwxMTMxLDExNDZdLCJhc3NpZ25lZSI6WyIxMDAxMDczIl19&id=275130&iterationID=1115&pId=0&type=REQUIREMENT)

#### Specified Reviewers:

/assign @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |    Approval management setup in personal workstation          |
| 🇨🇳 中文    |    个人工作台 审批          |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: bundle/review.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	"bytes"
+	"fmt"
+	"net/url"
+
+	"github.com/erda-project/erda/apistructs"
+	"github.com/erda-project/erda/bundle/apierrors"
+	"github.com/erda-project/erda/pkg/http/httputil"
+)
+
+func (b *Bundle) ListReviewApproval(orgID string, userID string, params url.Values) (*apistructs.GetReviewListResponse, error) {
```
Comment (sfwn) [inline bundle/review.go:27]: can you provide a better name? 
cc @Effet

**Review by sfwn — CHANGES_REQUESTED** [review]

File: bundle/review.go (hunk 1 above)
Comment (shuofan) [reply bundle/review.go]: updated bundle

File: modules/admin/component-protocol/components/personal-workbench/messageList/approval.go [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+	start, end := arrays.Paging(l.filterReq.PageNo, l.filterReq.PageSize, data.Total)
+	if start == -1 || end == -1 {
+		return
+	}
+	data.List = data.List[start:end]
+	return
+}
+
+func (l *MessageList) blockApproval(pageNo, pageSize uint64) (data []list.Item, userIDs []string, total int) {
```
Comment (sfwn) [inline modules/admin/component-protocol/components/personal-workbench/messageList/approval.go:50]: listBlockApprovals

File: modules/admin/component-protocol/components/personal-workbench/messageList/approval.go [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+					},
+				},
+			},
+		}
+		data = append(data, item)
+	}
+	return
+}
+
+func (l *MessageList) deployApproval(pageNo, pageSize uint64) (data []list.Item, userIDs []string, total int) {
```
Comment (sfwn) [inline modules/admin/component-protocol/components/personal-workbench/messageList/approval.go:112]: listDeployApproval

**Review by sfwn — CHANGES_REQUESTED** [review]

**Pull request merged**